test/index.test.ts

```diff
@@ -2,17 +2,23 @@
 // import index from '../src/index'
 
 import nock from "nock";
+import typeorm = require("typeorm");
+
 // Requiring our app implementation
 import myProbotApp from "../src";
 import { Probot } from "probot";
 
 // Requiring our fixtures
 import payload from "./fixtures/issues.ping.comment.json";
-import typeorm = require("typeorm");
+import pickUpPayload from "./fixtures/pulls.pickup.comment.json";
+
 const pongBody = { body: "pong! I am challenge bot." };
 const fs = require("fs");
 const path = require("path");
 
+// Mock the challenge-issue service
+jest.mock("../src/services/challenge-issue/index");
```

Comment:
Abstract the service as an interface and try to mock dependency injection.

Reply:
But now the implementation is based on the target class injection. If we use `probot.receive` API to mock receving the `pick-up` command, It seems we can't change the input parameters of pickUp method. Should we change the way of injection?

```
commands(app, "pick-up", async (context: Context) => {
    await pickUp(context, Container.get(ChallengeIssueService));
});
```

Reply:
I think you can mock the `Container.get`.